### ISSUE TITLE
.NET 9 Support

### ISSUE THREAD

**Issue by sjsanjsrh** [resonite-modding-group/ResoniteModLoader#19]
Beta 2024.12.16.1382
Runtime version: 9.0.0
Platform: Linux

The patch fails with the message System.PlatformNotSupportedException: CoreCLR version 9.0.0 is not supported.

**Comment by XDelta** [comment]
Waiting on harmony to support .NET 9 for the headless

**Comment by sjsanjsrh** [comment]
https://github.com/pardeike/Harmony/issues/635
Maybe harmony will be suppot soon.

**Comment by XDelta** [comment]
Yep, just waiting on a build. Will be included on releases as `0Harmony-Net9.dll`

**Comment by XDelta** [comment]
Still waiting on an official release, in the mean time, a build of harmony that appears to work under Net9 exists here https://github.com/stiefeljackal/pardeike.Harmony/releases/tag/temp-release

**Comment by badhaloninja** [comment]
This should be able to be closed right? The official net9 harmony release has come out

**Comment by XDelta** [comment]
Yes. Harmony 2.3.4+ support Net9 and a build is included as 
`0Harmony-Net9.dll`
